src/components/controls/Input.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { TextField } from '@material-ui/core';
+
+export default function Input(props) {
+
+    const { name, size, width, label, value,error=null, onChange, ...other } = props;
```

Comment:
destructure your props inside the parentheses 

example.

`export default const Input  =({ name, size, width, label, value,error=null, onChange, ...other })=>{}`